Discord.Net dependency not resolved

This error pops up multiple times when building & running the bot.
```
/home/nxse/NadekoBot/src/NadekoBot/NadekoBot.csproj : warning NU1603: NadekoBot.Core depends on Discord.Net (>= 2.0.0-beta-00882) but Discord.Net
 2.0.0-beta-00884 was not found. An approximate best match of Discord.Net 2.0.0-beta2-00884 was resolved.
```

Updating the dependency to the resolved version fixes the error.

--- a/NadekoBot.Core/NadekoBot.Core.csproj
+++ b/NadekoBot.Core/NadekoBot.Core.csproj
@@ -14,7 +14,7 @@
   <ItemGroup>
     <PackageReference Include="AngleSharp" Version="0.9.9.1" />
     <PackageReference Include="CommandLineParser" Version="2.2.1" />
-    <PackageReference Include="Discord.Net" Version="2.0.0-beta-00882" />
+    <PackageReference Include="Discord.Net" Version="2.0.0-beta2-00884" />
     <PackageReference Include="CoreCLR-NCalc" Version="2.1.3" />
     <PackageReference Include="Google.Apis.Urlshortener.v1" Version="1.32.2.138" />
     <PackageReference Include="Google.Apis.YouTube.v3" Version="1.32.2.1131" />

